File: Master/Web Services and Middleware Technologies/team-project-main/frontend/src/app/model/playlistDTO.ts
import {SongDTO} from "./songDTO";

export class PlaylistDTO {
  id!: string;
  name!: string;
  description!: string;
  songList!: SongDTO[];

  constructor(
    id: string,
    name: string,
    description: string,
    songList: SongDTO[]
  ) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.songList = songList;
  }
}

File: Master/Web Services and Middleware Technologies/individual-project-lauradeac-main/backend/src/main/java/com/backend/domain/dto/PlaylistDTO.java
package com.backend.domain.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Builder
public class PlaylistDTO {

    private String id;

    private String name;

    private String description;

    private List<SongDTO> songList;
}
